src/installation/index.md

```diff
@@ -6,44 +6,72 @@ Let's take a moment to discuss the Rust support for the different architectures
 
 [Ecosystem Overview]: ../overview/index.md
 
-## RISC-V targets
+## Rust installation
 
-The `RISC-V` architecture has support in the mainline Rust compiler so, the setup is relatively simple. There are two ways of proceeding with the installation:
-- Using the official Rust tools
-- Using [`espup`, a tool that will be covered later]
+To develop applications for ESP devices using Rust, you need to install the Rust compiler along with the appropriate toolchain and target(s). Depending on your device, it may be one of two architectures, each requiring a different setup.
 
-If you only want to use `RISC-V` targets, you can use the official Rust tools, for this approach we need [`rustup`] installed, and a [Rust nightly toolchain] with the `rust-src` [component]. We can install a nightly toolchain with the `rust-src` component via:
+If you have not yet installed Rust on your system, you can do so easily using [rustup]. For _macOS_ and _Linux_, it can be installed by running the following command:
 
 ```bash
-rustup toolchain install nightly --component rust-src
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
-These are the two recommended targets for most Espressif `RISC-V` chips:
-- For bare-metal (`no_std`) applications: `riscv32imc-unknown-none-elf`
-- For applications that require `std`: `riscv32imc-esp-espidf`
+For installation on Windows or alternative installation methods, refer to the instructions on the [rustup] website.
 
-> #### A note in RISC-V `no_std` Rust targets.
->
-> There are [different flavors of RISC-V 32 target in Rust] covering the different [RISC-V extensions].
+If you are [running Windows as your host operating system, you must also install one of the available ABIs]:
+- MSVC: This is the recommended ABI. When installing `rustup`, it will check if all the requirements are installed, and, if they are not, it will prompt the user to install them.
+- GNU: `rustup` does not check for the requirements, and it is expected that the user installs them properly.
 
+[rustup]: https://rustup.rs/
+[running Windows as your host operating system, you must also install one of the available ABIs]: https://rust-lang.github.io/rustup/installation/windows.html
 
-The bare-metal targets can be installed by running:
+## Rust with `std` runtime
 
-```bash
-rustup target add riscv32imc-unknown-none-elf
-```
+Regardless of the target architecture, to build a project using the [`std` approach], the following tools are required:
+- [`python`]: Required by ESP-IDF
+- [`git`]: Required by ESP-IDF
+- [`ldproxy`] crate: Simple tool to forward linker arguments given to [`ldproxy`] to the actual linker executable. To install it, use the following command:
+  - `cargo install ldproxy`
+- [ESP-IDF]: Espressif IoT Development Framework as it's used as our hosted environment.
+  - Users do not need to install ESP-IDf as it is automatically installed by [`esp-idf-sys`] (a crate that all `std` projects need to use).
```

Comment:
Do not list it, as the user does to install it. We could mention that it will be handled by esp-idf-sys but, out of the list